### ISSUE TITLE
Ui like in showcase video?

### ISSUE THREAD

**Issue by Yokopops** [KoljaB/RealtimeTTS#68]
Hello, I'm super interested in your project and have been messing around with AI of various types for a few months, however, I am relatively new to python and coding in general. 

This project stands out to me and I'm forcing myself to learn python and to be able to utilize it fully, due to myself being mute and unable to speak. I want to be able to use this to join in and talk to my friends over discord and have more of a presence.
 
short version, is it possible to share the simple UI (Or is it already there and I'm just dumb..) so that I may learn from it and expand on it?

Sorry, wasn't sure where else to ask this, Kindest regards.

**Comment by KoljaB** [comment]
The code I used for the video is this:
https://github.com/KoljaB/RealtimeTTS/blob/master/tests/pyqt6_speed_test.py

Also there's a new fastapi server, similar interface, streams to a webbrowser:
https://github.com/KoljaB/RealtimeTTS/tree/master/example_fast_api

And there's Linguflex, better interface and voice postprocessing:
https://github.com/KoljaB/Linguflex

**Comment by Yokopops** [comment]
Thanks for your reply, much appreciated! will check those out later. I hope to be able to make something to help other people in the same position as me someday.

Keep up the awesome work.

**Comment by rajibando** [comment]
I found the script `` in the User's Python Virtual directory. I executed it:
`$ python3 /(Home/UserDir)/RealtimeTTS-master/pyqt6_speed_test.py`

The following error message:
> Traceback (most recent call last):
  File "/(Home/UserDir)/RealtimeTTS-master/pyqt6_speed_test.py", line 1, in <module>
    from RealtimeTTS import (
  File "/(Home/UserDir)/Userpy3VirEnv/lib/python3.11/site-packages/RealtimeTTS/__init__.py", line 1, in <module>
    from .text_to_stream import TextToAudioStream  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/(Home/UserDir)/Userpy3VirEnv/lib/python3.11/site-packages/RealtimeTTS/text_to_stream.py", line 2, in <module>
    from .stream_player import StreamPlayer, AudioConfiguration
  File "/(Home/UserDir)/Userpy3VirEnv/lib/python3.11/site-packages/RealtimeTTS/stream_player.py", line 8, in <module>
    import resampy
ModuleNotFoundError: No module named 'resampy

Now what should be my next step regarding the module `resampy`?
